roseau/load_flow/models/lines/lines.py

```diff
@@ -273,12 +316,8 @@ def from_dict(
         )
 
     def to_dict(self) -> dict[str, Any]:
-        res = super().to_dict()
-        res.update(
-            {
-                "length": self.length,
-                "type_name": self.line_characteristics.type_name,
-                "type": "line",
-            }
-        )
-        return res
+        return {
+            **super().to_dict(),
+            "length": self.length,
+            "type_name": self.line_characteristics.type_name,
+        }
```

Comment:
/!\ Be careful with something that may happen. I note it here for later. 

The ground object is mandatory for line with shunt admittance. It means that the id of the ground must be serialized with the line in `to_dict` and read here with `from_dict`. Note that the ground will also be serialized with its list of connections. It means that the connection to the lines with shunt admittances may appear both in the format: one in the line serialization and one in the ground serialization. It may lead to confusion, that is why I recommend to `connect` the ground in the line constructor in such a way that this connection is not added in the `connections` field of the ground (or at least not serialized in the ground object).